zone/attack.cpp

```diff
@@ -4225,12 +4225,11 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 		//Note: if players can become pets, they will not receive damage messages of their own
 		//this was done to simplify the code here (since we can only effectively skip one mob on queue)
 		eqFilterType filter;
-		Mob* skip = attacker;
 		if (attacker && attacker->GetOwnerID()) {
 			//attacker is a pet, let pet owners see their pet's damage
 			Mob* owner = attacker->GetOwner();
 			if (owner && owner->IsClient()) {
-				if ((IsValidSpell(spell_id) || (FromDamageShield)) && damage > 0) {
+				if (FromDamageShield && damage > 0) {
```

Comment:
Shouldn't we at least leave the `IsValidSpell(spell_id)` check here and && the rest of the logic ?

Reply:
Yes!

Reply:
That's fine, the spell is not used in this call chain regardless. The code that was here before needs a lot of help